sw/device/silicon_creator/lib/boot_svc/boot_svc_next_boot_bl0_slot_unittest.cc

```diff
@@ -0,0 +1,42 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/silicon_creator/lib/boot_svc/boot_svc_next_boot_bl0_slot.h"
+
+#include "gtest/gtest.h"
+#include "sw/device/silicon_creator/lib/boot_svc/mock_boot_svc_header.h"
+#include "sw/device/silicon_creator/testing/rom_test.h"
+
+namespace boot_svc_next_boot_bl0_slot_unittest {
+namespace {
+
+class BootSvcNextBootBl0SlotTest : public rom_test::RomTest {
+ protected:
+  rom_test::MockBootSvcHeader boot_svc_header_;
+};
+
+TEST_F(BootSvcNextBootBl0SlotTest, ReqInit) {
+  boot_svc_next_boot_bl0_slot_req_t msg;
```

Comment:
nit: Let's zero-initialize this instead of leaving indeterminate so that we are 100 % sure that our code works.

```suggestion
  boot_svc_next_boot_bl0_slot_req_t msg{};
```